docker/compose.sh

```diff
@@ -50,23 +53,24 @@ else
     exit 1
 fi
 
+# Verify the image was built
+if ! docker image inspect linera >/dev/null 2>&1; then
+    echo "Failed to build linera image"
+    exit 1
+fi
+
 cd "$SCRIPT_DIR"
 
 set -xe
 
 # Create configuration files.
-# * Private server states are stored in `server.json`.
-# * `committee.json` is the public description of the Linera committee.
 linera-server generate --validators "$CONF_DIR/validator.toml" --committee committee.json --testing-prng-seed 1
 
 # Create configuration files for 10 user chains.
-# * Private chain states are stored in one local wallet `wallet.json`.
-# * `genesis.json` will contain the initial balances of chains as well as the initial committee.
-
```

Comment:
Why did you remove these comments? As far as I can tell, they're still accurate, no?